src/containers/CareerDetails.js

```diff
@@ -10,6 +11,7 @@ class CareerDetails extends Component {
   render() {
     return (
       <CareerContainer>
+        <NavBar />
```

Comment:
The Navbar is shared between all components on the app so each of these individual navbars should be moved to the `Router`.

Reply:
Here's a working example of a router:

```js
const Main = () => {
  return (
    <div>
      <NavBar />
      <Switch>
        <Route path="/Careers" exact component={Careers} />
        <Route path="/Careers/Liked" exact component={LikedCareers} />
        <Route path="/Careers/Details/:title" exact component={CareerDetails} />
        <Route path="/Settings" exact component={Settings} />
      </Switch>
    </div>
  );
};

const Routes = () => (
  <BrowserRouter>
    <Switch>
      <Route path="/AddCareer" exact component={AddCareer} />
      <Route path="*" component={Main} />
    </Switch>
  </BrowserRouter>
);
```

Reply:
Also see the above code snippet for more user-friendly URLs.

Reply:
So neat :sparkles: